packages/stripe-integration/src/stripe-upe/stripe-ocs-payment-strategy.ts

```diff
@@ -232,42 +229,32 @@ export default class StripeOCSPaymentStrategy implements PaymentStrategy {
             return;
         }
 
-        const titleFontSize = '15px'; // TODO: get style from theme
-        const titleFontWeight = '700'; // TODO: get style from theme
-        const titleColor = '#5f5f5f'; // TODO: get style from theme
         const radioColor = '#ddd'; // TODO: get style from theme
         const radioFocusColor = '#4496f6'; // TODO: get style from theme
-        const { radioIconOuterWidth, radioIconOuterStrokeWidth, radioIconInnerWidth } = style;
-        const radioIconSize = this._getRadioIconSizes(
-            radioIconOuterWidth,
-            radioIconOuterStrokeWidth,
-            radioIconInnerWidth,
-        );
+        const radioIconSize = this._getRadioIconSizes(style);
 
         return {
             variables: {
                 ...this.stripeUPEIntegrationService.mapAppearanceVariables(style),
-                fontFamily: 'Montserrat, Arial, Helvetica', // TODO: get style from theme
+                fontFamily: style.fontFamily?.toString(),
```

Comment:
Does Stripe SDK filters `undefined` and uses default values on their side?

Reply:
yes, Stripe has a default values, and this custom fonts it is only additionally loaded content inside their iframe